src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,160 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node head;
+    private Node tail;
+    private int size;
+
+    private class Node {
+        private T data;
+        private Node next;
+        private Node prev;
+
+        Node(T data, Node prev, Node next) {
+            this.data = data;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
     @Override
     public void add(T value) {
+        Node lastNode = tail;
+        Node newNode = new Node(value, lastNode, null);
+        tail = newNode;
+        if (lastNode == null) {
+            head = newNode;
+        } else {
+            lastNode.next = newNode;
+        }
+        size++;
     }
 
     @Override
     public void add(T value, int index) {
```

Comment:
Refactor this method next way:
![image](https://github.com/mate-academy/jv-linked-list/assets/20326858/168979e7-fa7e-4f87-875b-f1d1aacca790)
Create methods getNode and linkBefore
Use getNode in other places to avoid code repeating